### PR TITLE
irfan - week2 - test is passed

### PR DESCRIPTION
**Pull request by irfankaradeniz**

### CODE REVIEW

File: Week2/homework/maartjes-work.js [hunk 1]
```diff
@@ -46,8 +46,10 @@ const maartjesTasks = monday.concat(tuesday);
 const maartjesHourlyRate = 20;
 
 function computeEarnings(tasks, hourlyRate) {
-  // Replace this comment and the next line with your code
-  console.log(tasks, hourlyRate);
+  return tasks
+    .map(durations => durations.duration)
```
Comment (hasan-sh) [inline Week2/homework/maartjes-work.js]: `Multiply each duration by a per-hour rate for billing (use €20/hour) and sum it all up.` => first convert the duration to hour rate!
It's not wrong, but doing so makes it more clear!

File: Week2/homework/map-filter.js [hunk 2]
```diff
@@ -1,8 +1,7 @@
 'use strict';
 
 function doubleOddNumbers(numbers) {
-  // Replace this comment and the next line with your code
-  console.log(numbers);
+  return numbers.filter(number => number % 2 !== 0).map(number => number * 2);
```
Comment (hasan-sh) [inline Week2/homework/map-filter.js]: Is the name `number` in the map method the best name you could give??

>Think about what numbers you're filtering out in the first place!

**Comment by NoerGitKat** [comment]
Thanks for giving feedback @hasan-sh ! Hey @irfankaradeniz , could you take a look and implement the changes?

File: Week2/homework/maartjes-work.js (hunk 1 above)
Comment (irfankaradeniz) [reply Week2/homework/maartjes-work.js]: I got that, thanks!

File: Week2/homework/map-filter.js (hunk 2 above)
Comment (irfankaradeniz) [reply Week2/homework/map-filter.js]: I think it's not. Should it be oddNumber?

**Pull request closed without merging**